app/Http/Controllers/HomeController.php

```diff
@@ -75,9 +77,12 @@ public function showUser(Meeting $meeting){
         ->with('all_users', $all_users);
     }
 
-    public function showMeeting(Category $category){
+    public function showMeeting(Category $category, Meeting $meeting){
```

Comment:
since we don't use the `$meeting`, please remove it including the comments below.

Reply:
Okay, I understand. I removed $meeting and comments.